Packs/Polygon/Integrations/Polygon/Polygon.py

```diff
@@ -377,25 +411,51 @@ def get_monitor_indicators(report):
             end_time=p.get('exited_at'),
             path=p.get('filename'),
         )
-        ids.append(process)
+        command_results.append(CommandResults(
+            readable_output=get_human_readable_analysis_info(analysis_info),
+            outputs_prefix="Polygon.Analysis",
+            outputs_key_field="ID",
+            outputs=analysis_info,
+            indicator=process,
+            raw_response=res
```

Comment:
Please omit analysis_info output from the indicator's output, as it's already being returned elsewhere.
These CommandResults should be just for the indicator and its relevant human readable.